app/views/bulk_submissions/index.html.erb

```diff
@@ -1,29 +1,39 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-three-thirds">
```

Comment:
Isn't three thirds the whole width?

Reply:
don't know my govuk css classes, is that `govuk-grid-column-full` or `govuk-grid-column`?